camera_calibration_parsers/CMakeLists.txt

```diff
@@ -1,20 +1,23 @@
 cmake_minimum_required(VERSION 2.8)
 project(camera_calibration_parsers)
 
-find_package(catkin REQUIRED sensor_msgs rosconsole)
+find_package(catkin REQUIRED sensor_msgs rosconsole roscpp roscpp_serialization)
 
-find_package(Boost REQUIRED COMPONENTS filesystem)
+find_package(Boost REQUIRED COMPONENTS filesystem python)
+find_package(PythonLibs 2.7 REQUIRED)
+include_directories(include ${catkin_INCLUDE_DIRS} ${Boost_INCLUDE_DIRS} ${PYTHON_INCLUDE_DIRS})
 
-include_directories(include ${catkin_INCLUDE_DIRS} ${Boost_INCLUDE_DIRS})
+catkin_python_setup()
 
 catkin_package(
   INCLUDE_DIRS include
-  LIBRARIES ${PROJECT_NAME}
+  LIBRARIES ${PROJECT_NAME} ${PROJECT_NAME}_wrapper
```

Comment:
The wrapper library should not be here as it is not a proper library. Just leave that line unchanged.